src/main/java/com/internet/shop/service/UserService.java

```diff
@@ -13,4 +14,7 @@
     User update(User user);
 
     boolean delete(Long id);
+
+    Optional<User> findByLogin(String login);
 }
+
```

Comment:
```suggestion
```
at the end of file should be only one empty line

Reply:
done